groups/web-components.yml

```diff
@@ -0,0 +1 @@
+name: Web Components
```

Comment:
Interesting. I kinda think of web components as a composite feature: a web component is a distinct thing you can make or use, made up of smaller things you can use, rather than a group related-but-independent things.

See also: https://github.com/web-platform-dx/web-features/issues/971

Reply:
To be honest, I'm having a hard time getting a sense of the differences between a composite feature and a group.

Thinking more, I can see some differences:

- Groups don't get a status, they're mostly a way to navigate features. Composite features will get a status (once #971 is resolved).
- Groups can be very broad, like "html", whereas composite features remain at the individual thing a dev can use level.

If I picture myself navigating the list of features, I think I'd expect to find a group that's about web components somehow. Maybe that group should be the custom-elements group, available under `HTML > Elements > Custom elements`. But that wouldn't give me the list of everything I need to know in order to make a custom element. Shadow DOM isn't part of that group for example, nor is `<template>`.

Maybe I'm thinking about groups too much as a learning tool.

Maybe we try this web components group for now, and then re-evaluate once we have composite features?